### PR TITLE
New Element & Attribute KW's and docstring fixes

### PR DESCRIPTION
**Pull request by yahman72**
added KWs: 
- Element Attribute Should Match
- Element Value Should Be
- Get Elements

and fixed some docstrings

### CODE REVIEW

File: src/AppiumLibrary/version.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # -*- coding: utf-8 -*-
 
-VERSION = '1.3.4'
+VERSION = '1.3.4JNu'
```
Comment (jollychang) [inline src/AppiumLibrary/version.py]: 1.3.5？

Comment (yahman72) [inline src/AppiumLibrary/version.py]: oops, that was "my" version number.
yeah, it should be 1.3.5

Comment (jollychang) [inline src/AppiumLibrary/version.py]: please fix test

**Comment by yahman72** [comment]
hmmm, I have no idea why Travis fails:
- all unit tests pass on my PC
- I did not modify the failing part of the code

can you somehow re-execute Travis?

**Comment by jollychang** [comment]
you can try the detail log https://travis-ci.org/jollychang/robotframework-appiumlibrary/builds/85153184

**Comment by yahman72** [comment]
I did but the log doesn't make any sense - as said all tests pass on my PC

**Comment by jollychang** [comment]
I have fixed the test, thanks

**Pull request merged**